platform/mds/mds/src/main/java/org/motechproject/mds/annotations/EntityExtension.java

```diff
@@ -0,0 +1,24 @@
+package org.motechproject.mds.annotations;
+
+import javax.jdo.annotations.IdentityType;
+import javax.jdo.annotations.PersistenceCapable;
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * The <code>EntityExtension</code> annotation is used to point classes, that should be
```

Comment:
Looks like a copy-paste from the Entity javadoc. Also I think it is inaccurate as this annotation should be used for marking entities that extend already existent entities.